certbot-apache/certbot_apache/configurator.py

```diff
@@ -971,6 +972,72 @@ def enhance(self, domain, enhancement, options=None):
             logger.warn("Failed %s for %s", enhancement, domain)
             raise
 
+    def _enable_ocsp_stapling(self, ssl_vhost, unused_options):
+        """Enables OCSP Stapling
```

Comment:
Add a comment that this implicitly depends on SSLCertificateChainFile being set by other code.